docs/web_reference.rst

```diff
@@ -658,6 +658,10 @@ StreamResponse
       *Accept-Encoding* is not checked if *force* is set to a
       :class:`ContentCoding`.
 
+      *strategy* means zlib compression strategy.
+      Possible values are zlib.Z_DEFAULT_STRATEGY,
```

Comment:
```suggestion
      Possible values are :data:`zlib.Z_DEFAULT_STRATEGY`,
```

Reply:
(whether it's really "data" should be possible to check via intersphinx-untangled)

Reply:
Sorry, I can't find whether it is data or not here: https://webknjaz.github.io/intersphinx-untangled/docs.python.org/
zlib.Z_DEFAULT_STRATEGY is of type int, so should I use :data: here or not?

Reply:
Yes, use it but also add to `nitpick_ignore` in `docs/conf.py`.

Reply:
Done